### ISSUE TITLE
Garbled characters when using nvtop

### ISSUE THREAD

**Issue by yosider** [Syllo/nvtop#198]
When using nvtop on my Ubuntu server, some characters are garbled. Specifically, the frame lines are displayed with characters such as "l", "q", and "x", and "°C" is replaced with "fC" like this:

![image](https://user-images.githubusercontent.com/22992974/225315357-84efda1c-7e6f-4281-af1d-857b60ec7250.png)

When I previously upgraded the OS on this machine from Ubuntu 20.04 to 22.04, I accidentally interrupted the upgrade in the middle of the process, so some packages and settings may be missing.

If you have any idea of the cause, I would appreciate it if you could let me know.

**Comment by yosider** [comment]
After one night, I don't know why, but it was fixed!

**Comment by danmincu** [comment]
I have the same issue? any idea how to fix it?

**Comment by danmincu** [comment]
Answering my own question: the problem was isolated to Putty were despite having the translation in UTF-8 I had to modify the 
'Connection > Data > 'Terminal-type string' to 'putty' instead of 'xterm' - because [Putty ignores](https://stackoverflow.com/questions/54478320/activating-conda-environment-in-bash-script-that-runs-on-startup) the character sequence to switch to drawing-mode
![image](https://user-images.githubusercontent.com/21205717/230735963-77a78daf-49ca-4acb-b033-a71a58b311fc.png)

**Comment by Syllo** [comment]
Sorry for the delay.
@yosider it may have been a ncurses/ncursesw issue that got resolved after other software installs.
@danmincu, yes putty has its [own section in the README for this case :wink: ](https://github.com/Syllo/nvtop#troubleshoot).

**Comment by csaroff** [comment]
Yeah. Strangely it runs fine inside of tmux for me, but outside it looks garbled.
